src/System.Diagnostics.Process/src/PinvokeAnalyzerExceptionList.analyzerdata.uap

```diff
@@ -2,4 +2,3 @@ advapi32.dll!OpenProcessToken
 kernel32.dll!GetProcessAffinityMask
 kernel32.dll!SetProcessAffinityMask
 kernel32.dll!SetThreadAffinityMask
-ntdll.dll!NtQuerySystemInformation
```

Comment:
From what I understand, we had to have this line here before, because we were using this not-approved pinvoke in uap and to ignore the error indicating that. Now that the usage is being removed from uap, this line can be deleted.